### PR TITLE
misc test improvements

### PR DESCRIPTION
**Pull request by ValarDragon**
This PR contains the following misc test improvements:
* x/gamm: Deduplicate the FundAccounts coins constant
* x/gamm: Fix staticcheck being angry about unnecessary fmt.Sprintf's
* x/lockup: Fix noise from FundAccount in gas_test
* osmotestutils: Add GetFeeString function
* Replace complex fee cli line with simple osmotestutils call

## Description

This osmotestutils should accrue more improvements to testing here. The latency of trying to get improvements into the SDK, then back onto our own repo is far too high.

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

### CODE REVIEW

File: x/gamm/client/cli/cli_test.go [hunk 1]
```diff
@@ -78,7 +79,7 @@ func (s *IntegrationTestSuite) TestNewCreatePoolCmd() {
 		newAddr,
 		sdk.NewCoins(sdk.NewInt64Coin(s.cfg.BondDenom, 200000000), sdk.NewInt64Coin("node0token", 20000)), fmt.Sprintf("--%s=true", flags.FlagSkipConfirmation),
 		fmt.Sprintf("--%s=%s", flags.FlagBroadcastMode, flags.BroadcastBlock),
```
Comment (mattverse) [inline x/gamm/client/cli/cli_test.go:81]: Do we also want to deduplicate broadcast mode?

File: x/lockup/keeper/gas_test.go [hunk 2]
```diff
@@ -13,8 +14,13 @@ var (
...
+	err := simapp.FundAccount(suite.app.BankKeeper, suite.ctx, addr, coins)
+	suite.Require().NoError(err)
+	// start measuring gas
 	alreadySpent := suite.ctx.GasMeter().GasConsumed()
-	suite.LockTokens(addr, coins, dur)
+	_, err = suite.app.LockupKeeper.LockTokens(suite.ctx, addr, coins, dur)
+	suite.Require().NoError(err)
 	newSpent := suite.ctx.GasMeter().GasConsumed()
 	spentNow := newSpent - alreadySpent
 	return spentNow
```
Comment (mattverse) [inline x/lockup/keeper/gas_test.go:26]: Good catch on this! 
One further thing we can do in the future is set a range and simply test if the avgGas and maxGas is within the range, or something similar to remove hard coding gas, considering that there could be further instances where we would have to manually change the hard-coded gas whenever there is a logic change within the method.

**Review by mattverse — APPROVED** [review]
Looks LGTM!

File: x/gamm/client/cli/cli_test.go (hunk 1 above)
Comment (ValarDragon) [reply x/gamm/client/cli/cli_test.go:81]: thats a good idea, though I wonder if theres a better way we can just get more of these common flags to all be handled, to better mitigate the boilerplate. (E.g. Just appending all the common flags to the args array)

We should probs open an issue for tracking how to improve this

File: x/lockup/keeper/gas_test.go (hunk 2 above)
Comment (ValarDragon) [reply x/lockup/keeper/gas_test.go:26]: hrmm, thats a good idea. (Basically lower the sensitivity to small noise that we care about)

Depends on our general attitude towards gas, where does tracking small changes matter. I tend to agree with you, at this stage, having small aberrations is totally fine, as long as we can bound the max, and that max and average don't get too far apart.

**Pull request merged**